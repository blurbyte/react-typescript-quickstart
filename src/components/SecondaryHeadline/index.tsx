import styled from 'styled-components';

import { Color } from '@styles/colors';

const SecondaryHeadline = styled.h2`
  font-family: Arial, Helvetica, sans-serif;
  font-size: 1.4rem;
  color: ${Color.DarkGray};
  line-height: 1.2;
  margin: 1.4rem 0;
  margin-top: 3rem;
  padding-top: 1.6rem;
  border-top: 1px solid ${Color.Gray};
`;

export default SecondaryHeadline;
