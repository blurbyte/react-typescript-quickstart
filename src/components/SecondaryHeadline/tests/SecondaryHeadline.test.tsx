import * as React from 'react';
import { shallow } from 'enzyme';

import SecondaryHeadline from '../index';

describe('SecondaryHeadline', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<SecondaryHeadline />);
    expect(wrapper).toMatchSnapshot();
  });
});
