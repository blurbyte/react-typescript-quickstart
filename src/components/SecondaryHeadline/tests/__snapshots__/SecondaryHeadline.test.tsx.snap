// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SecondaryHeadline renders correctly 1`] = `
.c0 {
  font-family: Arial,Helvetica,sans-serif;
  font-size: 1.4rem;
  color: #444444;
  line-height: 1.2;
  margin: 1.4rem 0;
  margin-top: 3rem;
  padding-top: 1.6rem;
  border-top: 1px solid #908e89;
}

<h2
  className="c0"
/>
`;
